Dizzy/Screens/ReserveTable/ReserveTableVC.swift

```diff
@@ -0,0 +1,234 @@
+    }
+    
+    private func setupViews() {
+        commentsTextView.isHidden = true
+        makeCard()
+        setupTitleLabel()
+        setupMainmainStackView()
+        setupButtonsStackView()
+        setup(tonightButton, with: .tonight)
+        setup(tomorrowButton, with: .tomorrow)
+        setup(otherButton, with: .other)
+        setupSendButton()
+        setupCommentsTextView()
+        addGestures()
+        setupPlaceNameLabel()
+        setupCloseButton()
+        numberOfPeopleTextField.keyboardType = .numberPad
+    }
+    
+    private func setupTitleLabel() {
+        titleLabel.text = "Reserve a table".localized
+        titleLabel.textAlignment = .center
+        titleLabel.textColor = .primeryPurple
+    }
+    
+    private func setupMainmainStackView() {
+        mainStackView.alignment = .fill
+        mainStackView.axis = .vertical
+        mainStackView.spacing = Metrics.doublePadding
+    }
+    
+    private func setupButtonsStackView() {
+        buttonsStackView.alignment = .fill
+        buttonsStackView.axis = .horizontal
+        buttonsStackView.spacing = Metrics.doublePadding
+    }
+    
+    private func setupSendButton() {
+        sendButton.setTitle("SEND".localized, for: .normal)
+        sendButton.titleLabel?.font = Fonts.h10(weight: .bold)
+        sendButton.setTitleColor(.white, for: .normal)
+        sendButton.layer.cornerRadius = signInCornerRadius
+        sendButton.backgroundColor = signInButtonBackgroundColor
+        sendButton.addTarget(self, action: #selector(onSendPressed), for: .touchUpInside )
+    }
+    
+    private func setupCloseButton() {
+        closeButton.setImage(Images.whiteBackButton(), for: .normal)
+        closeButton.addTarget(self, action: #selector(onCloasePressed), for: .touchUpInside )
+    }
+    
+    private func setup(_ button: UIButton, with buttonType: ButtonType) {
+        button.tag = buttonType.rawValue
+        button.setTitle(buttonType.title, for: .normal)
+        button.setTitleColor(.blue, for: .normal)
+        button.addTarget(self, action: #selector(buttonPressed(_:)), for: .touchUpInside)
+    }
+    
+    private func setupPlaceNameLabel() {
+        placeNameLabel.text = viewModel.placeName
+        placeNameLabel.textAlignment = .center
+        placeNameLabel.textColor = .white
+        placeNameLabel.font = Fonts.h1(weight: .bold)
+    }
+    
+    private func setupCommentsTextView() {
+        commentsTextView.font = Fonts.h10()
+        commentsTextView.textContainer.maximumNumberOfLines = 2
+        commentsTextView.textContainer.lineBreakMode = .byWordWrapping
+        commentsTextView.setBorder(borderColor: .lightGray)
+    }
+    
+    private func layoutViews() {
+        mainStackView.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(Metrics.padding)
+            make.leading.equalToSuperview().offset(Metrics.doublePadding)
+            make.trailing.equalToSuperview().inset(Metrics.doublePadding)
```

Comment:
are you sure `inset` is not expanding the view? if do, dont use it

Reply:
inset is qual to offset with minus number.